.github/workflows/tests.yml

```diff
@@ -46,29 +46,24 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.8", "3.11"]
+        test-env: ["dev-py38", "dev-py39", "dev-py310", "dev-py311", "dev-py312"]
         os: ["macos-latest", "ubuntu-latest", "windows-latest"]
 
     steps:
       - uses: actions/checkout@v3
 
       - uses: prefix-dev/setup-pixi@v0.3.0
         with:
-          pixi-version: v0.6.0
+          pixi-version: v0.17.0
           cache: true
```

Comment:
```suggestion
          cache: true
          environments: ${{ matrix.test-env }}
```
![image](https://github.com/conda-incubator/conda-auth/assets/29506042/44edaf62-601d-441d-8083-60debd205b0a)
https://pixi.sh/latest/advanced/github_actions/#multiple-environments

Reply:
also, `cache: true` is the default nowadays (you need to update setup-pixi to 0.5.1 anyway